README.md

```diff
@@ -1,31 +1,69 @@
 
 #### Description
-This extension is inside the OpenAPI info object and contains basic API-level metadata. Currently...
+This extension is inside the OpenAPI info object and contains basic API-level metadata. Currently, there are two (required) properties: component (KP, ARA, ARS, or Utility) and team (one or more Translator team names). Current tags include:
+
+- **infores:** Information Resource CURIE identifying the Translator Resource
+- **component:** One of  "KP", "ARA", "ARS", "Utility"
+- **team**: List of contributing Translator teams (see [schema file](https://github.com/NCATSTranslator/translator_extensions/blob/main/x-translator/smartapi_x-translator_schema.json) for the full list)
+- **biolink-version:**  One SemVer-formatted version number (e.g. '1.8.2', '2.2.16', etc.)
+
+Associated rules:
+ 
+- if the API is tagged "translator" in the OpenAPI top-level tags, then the x-translator extension must be present (in OpenAPI info)
+- if x-translator is present (in OpenAPI info), then the API must be tagged "translator" in the OpenAPI top-level tags  
 
-- there are two (required) properties: component (KP, ARA, ARS, or Utility) and team (one or more Translator team names)
-- there are rules:  
-  - if the API is tagged "translator" in the OpenAPI top-level tags, then the x-translator extension must be present (in OpenAPI info)
-  - if x-translator is present (in OpenAPI info), then the API must be tagged "translator" in the OpenAPI top-level tags  
+#### Examples
 
-#### Examples  
 An example of a valid x-translator extension in YAML is:  
 ```
   tags:
   - name: translator    ## required translator tag
   - name: biothings
   info:
     x-translator:      ## required x-translator extension
+      infores: "infores:biolink-api"
       component: KP
       team:
       - "Service Provider"
+      biolink-version: "2.2.16"
 ```
-This same example in JSON and other examples of valid and invalid instances (in JSON) are in [this file](https://github.com/NCATSTranslator/translator_extensions/blob/main/x-translator/smartapi_x-translator_examples.txt). 
+This same example in JSON and other examples of valid and invalid instances (in JSON) are in [this example file](https://github.com/NCATSTranslator/translator_extensions/blob/main/x-translator/smartapi_x-translator_examples.txt). 
 
 ## x-trapi
-This extension's schema has not been written / registry files do not contain this extension. It will likely contain API- and operation-level metadata that TRAPI expects.
\ No newline at end of file
+
+#### Status
+
+This extension's schema has been started, and registry files currently contain this extension.
+
+#### Description
+
+This extension is inside the OpenAPI info object and contains basic API-level metadata. Currently, there are three (required) properties: version, operations and test_data_location, plus other optional ones:
+
+- **version:** One TRAPI version SemVer formatted number (e.g. '1.1', '1.2', '1.3')
```

Comment:
Give more info? "States the TRAPI standard version that the Translator Resource is compliant with / built to. The TRAPI standard and its versions can be viewed here".